db/migrate/20230720131714_add_fields_to_long_read_qc_results.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# Adds new optional fields to long_read_qc_result
+class AddFieldsToLongReadQcResults < ActiveRecord::Migration[7.0]
+  def change
+    change_table :long_read_qc_result, bulk: true do |t|
+      t.string :priority_level, comment: 'Priority level eg Medium, High etc'
```

Comment:
Again @emrojo I think we need to check that these are relevant.

Reply:
Update from meeting 27/07: date_required_by and reason_for_priority are no longer required and can be removed from this schema. priority_level is not a qc_result attribute but should be moved to sample.

Reply:
Done. Added the migration to add priority_level to sample table.